actions/auth.go

```diff
@@ -32,13 +26,34 @@ import (
 	googleoauth2 "google.golang.org/api/oauth2/v2"
 	"google.golang.org/api/option"
 	"gopkg.in/ezzarghili/recaptcha-go.v3"
+
+	"github.com/mailbadger/app/emails"
+	"github.com/mailbadger/app/entities"
+	"github.com/mailbadger/app/entities/params"
+	"github.com/mailbadger/app/logger"
+	"github.com/mailbadger/app/storage"
+	"github.com/mailbadger/app/utils"
+	"github.com/mailbadger/app/validator"
 )
 
 // PostAuthenticate authenticates a user with the given username and password.
 func PostAuthenticate(c *gin.Context) {
-	username, password := c.PostForm("username"), c.PostForm("password")
+	body := &params.PostAuthenticate{}
+	if err:=c.ShouldBind(body); err != nil {
+		logger.From(c).WithError(err).Error("Unable to bind authentication params.")
```

Comment:
@Gligor23 i think we don't need log for this. also on validate.

Reply:
Maybe we should have a lower level log, or is this something not worth logging?

Reply:
What is the error here in case ShouldBind fails?

Reply:
After discussion, we should remove the logs from ShouldBind and Validate. Resolving this.